event.go

```diff
@@ -20,12 +20,13 @@ var eventPool = &sync.Pool{
 // Event represents a log event. It is instanced by one of the level method of
 // Logger and finalized by the Msg or Msgf method.
 type Event struct {
-	buf   []byte
-	w     LevelWriter
-	level Level
-	done  func(msg string)
-	stack bool   // enable error stack trace
-	ch    []Hook // hooks from context
+	buf                  []byte
+	w                    LevelWriter
+	level                Level
+	done                 func(msg string)
+	stack                bool   // enable error stack trace
+	ch                   []Hook // hooks from context
+	callerSkipFrameCount int    // The number of additional frames to skip when printing the caller.
```

Comment:
That's very long, what about `skipFrame`?

Reply:
Sounds good.  Done.  3d15cd96b297c50fb0e980abbf94883c597572d7